spring-boot-starter/starter-client/spring-boot/src/main/java/org/camunda/bpm/client/spring/boot/starter/impl/properties/BasicAuthProperties.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.client.spring.boot.starter.impl.properties;
+
+public class BasicAuthProperties {
+
+  protected boolean enabled = false;
```

Comment:
With regards to other starters that we feature, we usually offer "configured == enabled" and "not configured == disabled" for authentication. I do not see a use case for this flag to be honest since you can also comment the configured authentication in the properties, so I would vote to remove it.
If we remove it, we need to make sure we handle them correctly in `PropertiesAwareClientFactory` in case of the properties not being defined in the `application.yml`

Reply:
👍